### PR TITLE
Addd note in README to use lock file if `npm i` fails

### PR DESCRIPTION
**Pull request by jtamsut**
We aren't checking in the `package-lock.json` file for the workspace. This poses a problem if new versions of packages are broken. To mitigate this unlikely scenario we have included in a `package-lock.json` file in the `script` directory that can be used.

### CODE REVIEW

**Review by felipesu19 — DISMISSED** [review]

File: docs/development.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ npm i
 npm run build -ws
 ```
 
+**Note**: We have included a `package-lock.json` in `script/workspace`. If packages cannot be installed correctly re-run `script/boostrap` and run `npm ci` to
```
Comment (lrotschy) [inline docs/development.md:34]: Feel free to ignore. Just a thought but it might be too wordy.

```suggestion 
**Note**: We have included a `package-lock.json` in `script/workspace`. If `npm run build -ws` fails because packages are not installed correctly with `nmp i`, re-run `script/boostrap` and run `npm ci` to
```

**Review by lrotschy — APPROVED** [review]
